corehq/apps/reports/static/reports/js/bootstrap3/hq_report.js

```diff
@@ -21,14 +21,15 @@ hqDefine("reports/js/bootstrap3/hq_report", [
         self.datespan = options.datespan;
         self.filterSet = options.filterSet || false;
         self.needsFilters = options.needsFilters || false;
-        self.filterAccordion = options.filterAccordion || "#reportFilters";
-        self.filterSubmitSelector = options.filterSubmitSelector || '#paramSelectorForm button[type="submit"]';
+        self.htmlIDSuffix = options.html_id_suffix;
+        self.filterAccordion = options.filterAccordion || "#reportFilters" + self.htmlIDSuffix;
```

Comment:
Will [this logic](https://github.com/dimagi/commcare-hq/blob/891cc2068463782bb606b2699eced654627fca8f/corehq/apps/reports/static/reports/js/bootstrap3/hq_report.js#L151-L166) work when there are multiple accordions and buttons? It looks like it might open all of the accordions and update all of the buttons (although maybe that's fine, it also looks like this is only called once, on page init).